src/charactersheet/models/common/treasure.js

```diff
@@ -20,21 +20,26 @@ export function Treasure() {
     self.copper = ko.observable(0);
 
     self.worthInGold = ko.computed(function(){
-        var parsedPlatinum = parseInt(self.platinum()) || 0;
-        var parsedGold = parseInt(self.gold()) || 0;
-        var parsedElectrum = parseInt(self.electrum()) || 0;
-        var parsedSilver = parseInt(self.silver()) || 0;
-        var parsedCopper = parseInt(self.copper()) || 0;
+        const parsedPlatinum = parseInt(self.platinum()) || 0;
+        const parsedGold = parseInt(self.gold()) || 0;
+        const parsedElectrum = parseInt(self.electrum()) || 0;
+        const parsedSilver = parseInt(self.silver()) || 0;
+        const parsedCopper = parseInt(self.copper()) || 0;
 
-        var adjPlatinum = parsedPlatinum * 10;
-        var adjGold = parsedGold;
-        var adjElectrum = parsedElectrum / 2;
-        var adjSilver = parsedSilver / 10;
-        var adjCopper = parsedCopper / 100;
+        const copperToSilver = Math.floor(parsedCopper / 10);
```

Comment:
Would it be better to `floor` it at the last possible calculation? If not, you're constant losing bits of currency. Then again, not sure if you would ever save enough fractions of a coin to make it count. Thoughts?

Reply:
Yeah I don't think it matters. No reason to roll up the floats. Even if you had one missing for each currency it would never round up. For example, 9 copper, 9 silver, 1 electrum does not equal 1 gold. Floating points shouldn't be used in currency calculations IMO.